src/main/java/app/model/SupplyFruit.java

```diff
@@ -0,0 +1,52 @@
+package app.model;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class SupplyFruit {
```

Comment:
```suggestion
public class SupplyFruitBatch {
```
it would be more logical, cause fruit doesn't have quantity